test_docker.sh

```diff
@@ -1 +0,0 @@
-docker run --rm -v $(pwd):/Qt.py mottosso/qt.py27
```

Comment:
Because both use the same image, I think it's safe to include both commands in this file now. Before, I only opted for one image, as I didn't want to build both images just to run simple tests. I left that to Travis.